ArjunaCore/arjunacore/pom.xml

```diff
@@ -41,6 +41,7 @@
         <artifactId>build-helper-maven-plugin</artifactId>
         <executions>
           <execution>
+            <?m2e execute onConfiguration?>
```

Comment:
@tomjenkinson I've not seen this xml syntax before - what does it do?

Reply:
It allows the IDE I am wanting to work with to actually add the listed folders as source folders and thus allows the IDE to resolve certain files without showing syntax errors. It stands (I think) roughly for "Maven 2 Eclipse". There is some reference here: https://eclipse.dev/m2e/documentation/release-notes-17.html#new-syntax-for-specifying-lifecycle-mapping-metadata.

Reply:
Interesting link, thanks. I'll resolve this conversation then.